Mastodon/Scene/Onboarding/ServerRules/MastodonServerRulesViewController.swift

```diff
@@ -38,34 +113,18 @@ extension MastodonServerRulesViewController {
         
         setupOnboardingAppearance()
         defer { setupNavigationBarBackgroundView() }
-        
-        tableView.translatesAutoresizingMaskIntoConstraints = false
-        view.addSubview(tableView)
-        tableView.pinToParent()
 
-        tableView.delegate = self
-        viewModel.setupDiffableDataSource(tableView: tableView)
-        
         navigationItem.largeTitleDisplayMode = .always
-
-        navigationItem.rightBarButtonItem = UIBarButtonItem(title: L10n.Scene.ServerRules.Button.confirm, style: .done, target: self, action: #selector(MastodonServerRulesViewController.nextButtonPressed(_:)))
         title = L10n.Scene.ServerRules.title
     }
-    
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
-        
-        tableView.flashScrollIndicators()
-    }
-    
 }
 
 extension MastodonServerRulesViewController {
-    @objc private func backButtonPressed(_ sender: UIButton) {
+    @objc private func backButtonPressed(_ sender: UIButton?) {
         navigationController?.popViewController(animated: true)
     }
 
-    @objc private func nextButtonPressed(_ sender: UIButton) {
+    @objc private func nextButtonPressed(_ sender: UIButton?) {
```

Comment:
Why is this button `Optional` now?

Reply:
The selector's target _is_ always an optional value as the target might have been deallocated once this code is reached. Previously we have implicitly unwrapped it. However good point, we can completely remove the target attribute as we've never used it anyways.

Ref: https://developer.apple.com/documentation/uikit/uibarbuttonitem/1617127-action